.travis.yml

```diff
@@ -26,7 +26,9 @@ install:
 
 before_script:
   - export DATABASE_URL='postgres://postgres@localhost/warp'
+  - export DJANGO_SETTINGS_MODULE=config.settings.test
   - psql -c 'create database warp;' -U postgres
+  - python manage.py makemigrations && python manage.py migrate
```

Comment:
We have to make migrations of 'presentation' app too, don't we?
https://docs.djangoproject.com/en/1.11/ref/django-admin/#makemigrations